Makefile

```diff
@@ -8,6 +8,6 @@ GO_BUILD_PACKAGES :=./pkg/...
 # You can list all the golang related variables by:
 #   $ make -n --print-data-base | grep ^GO
 
-include vendor/github.com/openshift/library-go/alpha-build-machinery/make/golang.mk
+include vendor/github.com/openshift/build-machinery-go/make/golang.mk
```

Comment:
these changes don't belong to bump commit

Reply:
separated into separate commit